src/mlpack/methods/ann/init_rules/he_init.hpp

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file he_init.hpp
+ * @authors Dakshit Agrawal and Prabhat Sharma
```

Comment:
Can you split up the author section (for each author another line that starts with ``@author``), that way doxygen can pick this up.